content/pages/enterprise.html

```diff
@@ -0,0 +1,132 @@
+  <body>
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/homepage.html" as homepage %}
+
+{% macro about_divide(text=None) %}
+<div class="ui text container">
+  <div class="ui {%- if text %} horizontal {%- endif %} divider">
+    {% if text %}{{ text }}{% endif %}
+  </div>
+</div>
+{% endmacro %}
+
+  <section>
+    {% call basic.header_lead(
+    header="Read the Docs is the <em><strong>all-in-one</strong></em> solution you're looking for your documentation")
+    %}
+    <p>
+        We offer customized enterprise plans for organizations,
+        and can support many different customizations.
+        Works with our team to find the plan that's right for you.
+    </p>
+      <a class="ui center aligned padded segment button" href="/pricing/enterprise/#enterprise-inquiry">Get started on an Enterprise plan</a>
+      </div>
+    {% endcall %}
+  </section>
+
+
+
+
+<section id="additional-features">
+
+    {{ about_divide("Enterprise Features") }}
+  
+    <div class="ui very padded container" id="more">
+      <div class="ui vertically padded centered grid">
+  
+        {% call homepage.feature_detail_item(header="Custom builders", icon="fad fa-robot") %}
+            Get additional memory or CPU in your builds,
+            to support your larger documentation builds.
+        {% endcall %}
+  
+        {% call homepage.feature_detail_item(header="Support SLA", icon="fad fa-folder-tree") %}
+        Get guaranteed support response times for your most important documentation projects. 
+        {% endcall %}
+  
+        {% call homepage.feature_detail_item(header="Custom SSO", icon="fad fa-shield-keyhole") %}
+        If you have SSO needs outside of our Git integrations,
+        we can integrate with custom SSO providers to keep your docs secure.
+        {% endcall %}
+
+        {% call homepage.feature_detail_item(header="Custom build images", icon="fad fa-signs-post") %}
+          Does your documentation have specific build requirements?
+          We can provide custom Docker images for your builds to keep them fast and reliable.
+        {% endcall %}
+  
+        {% call homepage.feature_detail_item(header="Additional Audit Tracking", icon="fad fa-signs-post") %}
+            If you need longer term storage for your Audit Logging and Analytics information,
+            we can provide that.
+            This is great for compliance requirements.
+        {% endcall %}
+
+        {% call homepage.feature_detail_item(header="Custom Requirements", icon="fad fa-signs-post") %}
+          Have additional needs that we haven't addressed here?
+          We're happy to work with you to provide additional functionality as required for your documentation solution.
+        {% endcall %}
+  
+      </div>
+    </div>
+  </section>
+
+
+  {# Enterprise customer callout #}
+  <section id="enterprise-inquiry">
+    <div class="ui very padded text container">
+      <div class="ui inverted center aligned padded segment">
```

Comment:
The from looks more natural left aligned, I'd drop the center alignment here but keep the header center alignment if needed.

```suggestion
      <div class="ui inverted padded segment">
```